cost-analyzer/values.yaml

```diff
@@ -839,6 +839,10 @@ kubecostDeployment:
   #   type: RollingUpdate
   queryServiceReplicas: 0
   queryService:
+    resources: 
+      requests:
+        cpu: 1000m
+        memory: 25Gi
```

Comment:
@srpomeroy Thoughts on leaving `resource.requests` commented out? I'm unsure whether these defaults are appropriate for all QSR users.

Reply:
@thomasvn 
The thought was that if you need the QSR, you have a large dataset and we are recommending a relatively large starting point. Perhaps 500Mi is better?
Perhaps a comment:

# You can use the Kubecost savings report for 'Right-size your container requests' to determine the recommended the resource requests once the pod has run for 24 hours.

Reply:
Adjusted mem requests down and added a note. Are you good with this approach @thomasvn?

Reply:
@srpomeroy One last thing. Do you think should we also bring down the default CPU requests in proportion with the MEM requests?

Also could you share results of `helm template` in the PR's comments? After that I'm good to approve!